src/Employer/Employer.Web/Configuration/ConfigurationExtensions.cs

```diff
@@ -89,7 +89,7 @@ public static void AddMvcService(this IServiceCollection services, IHostingEnvir
 
                 opts.AddTrimModelBinderProvider(loggerFactory);
             })
-            .AddFluentValidation(fv =>fv.RegisterValidatorsFromAssemblyContaining<Startup>())  
+            .AddFluentValidation()
```

Comment:
Hi Chris, Just confirming if the validations are still working?

Reply:
Have just checked Employer. This is still working.